Library/Formula/libbluray.rb

```diff
@@ -19,13 +19,17 @@ class Libbluray < Formula
     depends_on "ant" => :build
   end
 
+  if OS.linux?
+    depends_on "libxml2" => :build
+  end
```

Comment:
Use `depends_on "libxml2" => :build if OS.linux?` and place it after the existing depends. Are you sure the dependency is `:build` only? Does it not require the libraries to be installed to run?